AEPServices/Sources/dataqueue/SQLiteWrapper.swift

```diff
@@ -75,7 +75,7 @@ internal struct SQLiteWrapper {
     ///   - database: the database connection
     ///   - sql: the SQL statement
     /// - Returns: an `Optional` result of a database  query
-    static func query(database: OpaquePointer, sql: String) -> [[String: String]]? {
+    public static func query(database: OpaquePointer, sql: String) -> [[String: String]]? {
```

Comment:
We had some concerns around this APIs when we previously discussed making this class public. Is it possible to represent the return values better instead of String?

Reply:
i'm open to suggestions as long as it doesn't break backwards compatibility with our internal code that uses the wrapper.  

did you have a specific idea on what could be returned other than an array of dictionaries (each dictionary representing a record from the database)?